README.md

```diff
@@ -55,7 +55,8 @@ The BARK ecosystem is composed of multiple components that all share the common
 
 *   [BARK-ML](https://github.com/bark-simulator/bark-ml/): Machine learning library for decision-making in autonomous driving.
 *   [BARK-MCTS](https://github.com/bark-simulator/planner-mcts): Integrates a template-based C++ Monte Carlo Tree Search Library into BARK to support development of both single- and multi-agent search methods.
-*   [BARK-Rules-MCTS](https://github.com/bark-simulator/planner-rules-mcts): Integrates traffic rules within Monte Carlo Tree Search with lexicographic ordering.
+*   [BARK-Rules-MCTS](https://github.com/bark-simulator/planner-rules-mcts): Integrates traffic rules within Monte Carlo Tree Search with lexicographic ordering. 
+*   [BARK-MIQP](https://github.com/bark-simulator/planner-miqp): MINIVAN Planner based on MIQP for single- and multi-agent planning. To use it, comment in the planner_miqp repository in the WORKSPACE file. Then, build it via `bazel build //... --define planner_miqp=true  --test_env LD_LIBRARY_PATH=/opt/ibm/ILOG/CPLEX_Studio1210/opl/bin/x86-64_linux`, where the latter specifies the path to your CPLEX installation.
```

Comment:
I would leave out the build instructions. Maybe rather provide an example that you can run? `bazel run //examples/:minivan`